pom.xml

```diff
@@ -0,0 +1,157 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I do not see the following things:
Forms for CRUD operations (POST/PUT/DELETE)
2.Store recent user selecitons in session
3.Log user browser when some app functionality is accessed
4.Render 2 different views depending on value of Alternates header
5.Use content negotiation with message formatting for rendering XML or JSON
6.Implement PDF upload/download functionality